packages/sources/dxfeed-test/src/endpoint/price.ts

```diff
@@ -0,0 +1,65 @@
+import { BatchWarmingTransport } from '@chainlink/external-adapter-framework/transports/batch-warming'
+import { makeLogger } from '@chainlink/external-adapter-framework/util'
+import { EndpointTypes } from './price-router'
+
+const logger = makeLogger('DxFeed Price Batched')
+
+const quoteEventSymbols: { [key: string]: boolean } = {
+  'USO/USD:AFX': true,
+}
```

Comment:
While we're at it, should we split this out into a JSON file along with overrides?

Reply:
do you mean to have it in separate JSON file and import it ?

Reply:
Correct